### PR TITLE
Parameterised bucket so that we can move to deployTools

### PR DESCRIPTION
**Pull request by philwills**
This has done a successful test run, but I'm unsure how it's regularly triggered.

Whilst I've tried to maintain some backwards compatibility with the old methods, I'm not sure that's really possible, so I'd appreciate opinions on the best migration approach.

### CODE REVIEW

**Comment by philmcmahon** [comment]
Looks good! Maybe for the migration we could get this to post to both S3 buckets, then switch off the lambda in the composer account?

**Comment by philmcmahon** [comment]
It's probably worth deleting the circle.yml file and riffraff config to stop future contributors from getting confused! Though I could do this in a separate PR.

File: cloudformation/cfn.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        PolicyDocument:
+          Statement:
+          - Effect: Allow
+            Action:
+            - s3:PutObject
+            Resource: !Sub arn:aws:s3:::${TargetBucket}/*
+          - Effect: Allow
+            Action:
+            - dynamodb:Scan
+            Resource: !Sub arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${ConfigTable}
```
Comment (philwills) [inline cloudformation/cfn.yaml:40]: I still don't understand why you can't get this in a sane way, but `Sub` certainly makes it less horrid.

**Comment by philwills** [comment]
I'm tempted to merge this now, even though there's not a good solution for the old bucket. Does that sound reasonable?

**Comment by philmcmahon** [comment]
👍 
Yep, we can leave the other one running for now then I could eventually deploy this version in the composer account...

**Pull request merged**